README.md

```diff
@@ -54,6 +54,32 @@ import * as Turbo from '@hotwired/turbo'
 +TurboPower.initialize(Turbo.StreamActions)
 ```
 
+### Installation on a stock Rails 7 install, which utilizes [importmaps](https://github.com/rails/importmap-rails)
+
+On a stock install of Rails 7 modify the following files:
+
+**config/importmaps.rb**
+```diff
+pin "application", preload: true
+- pin "@hotwired/turbo-rails", to: "turbo.min.js", preload: true
++ pin "@hotwired/turbo", to: "turbo.min.js", preload: true
```

Comment:
I don't think this should be necessary. It should also be possible to also utilize the `turbo-rails` package. 

So I think we can keep this as `turbo-rails` and just adjust the import in the `application.js`